go.mod

```diff
@@ -4,14 +4,14 @@ go 1.18
 
 require (
 	github.com/T-Systems-MMS/tnd v0.0.0-20230404085219-434ecc2a5379
+	github.com/fsnotify/fsnotify v1.6.0
```

Comment:
nit-pick: this could be moved to a new PR with the other cleanups that were previously in this PR.